src/main/java/com/smalaca/messagesender/repository/inmemory/InMemoryUserRepository.java

```diff
@@ -58,10 +58,10 @@ public boolean blockUser(String login) {
     }
 
     @Override
-    public void updateUser(UserDto userDto) {
-        User user = getUserByLogin(userDto.getLogin());
-        if (new UserFactory().createFrom(userDto).equals(user))
+    public void updateUser(User user) {
+        if (user.equals(getUserByLogin(user.getLogin()))) {
             throw new UserAlreadyExistException(user.getLogin());
```

Comment:
Ale to przecież powinno być odwrotnie, czyli:
- mogę robić update, gdy użytkownik istnieje
- wyrzucam wyjątek, gdy nie ma czego update'ować

**#bug**